### DISCUSSION TITLE
File format supported

### DISCUSSION THREAD

**Discussion by mathieulinares** [scanberg/viamd#36] [General]
# Supported file formats for topologies:
• gro - Gromacs (ASCII, any precision)
• pdb - Protein Databank (ASCII, reduced precision)
• cif - PDBx/mmcif (ASCII) [Not thoroughly tested]
• xyz/arc/xmol - XYZ formats in variations (ASCII)
• LAMMPS data file

The currently supported file formats for trajectories:
• xtc - Gromacs (binary, compressed, full precision)
• trr - Gromacs (binary, full precision)
• pdb - Protein Databank, frame encoded via MODEL/TER (ASCII, re-
duced precision)
• xyz/arc/xmol - XYZ formats in variations (ASCII, any precision)
• LAMMPS ASCII trajectory
• dcd - NAMD trajectory

# Supported file formats for script import:
• edr - Gromacs Energy (binary)
• xvg - Grace Tabular (ASCII)
• csv - Comma Separated Values (ASCII)

# Supported file formats for script export:
• xvg - Grace Tabular (ASCII)
• csv - Comma Separated Values (ASCII)
• cube - Cube file, volume + structure (ASCII)

In the future, we aim to add support for CHARMM, and AMBER.

# Supported file formats for Electronic Structure calculations:
• h5 - VeloxChem output file

We will support TREX format in the future. We encourage developers and users of other electronic structure code to develop parsers so VIAMD can read other format and are available to help in the process.

**Comment by swjackson286** [comment]
Any chance .dcd will be added? Is there any way I can load my .dcd trajectory file?

**Comment by mathieulinares** [comment]
Hi. 
We are thinking about adding dcd support in the future but do not have it scheduled yet. 
In the meantime mdtraj is perfect to convert your trajectory to xtc.

**Reply by swjackson286** [reply]
Thanks. And sorry--just now seeing the part above about adding support for NAMD later. Thanks again!

**Reply by mathieulinares** [reply]
Now dcd support has been added.
